src/test/java/org/apache/maven/shared/verifier/VerifierTest.java

```diff
@@ -130,4 +139,74 @@ void testDefaultFilterMap() throws VerificationException
         assertTrue( filterMap.containsKey( "@basedir@" ) );
         assertTrue( filterMap.containsKey( "@baseurl@" ) );
     }
+
+    @Test
+    void testDefaultMavenArgument() throws VerificationException
+    {
+        TestVerifier verifier = new TestVerifier( "src/test/resources" );
+
+        verifier.executeGoal( "test" );
```

Comment:
Unrelated: We see now that the `executeGoal()` name is misleading because `test` isn't a goal, it is a phase....

Reply:
I saw it ... I thinking to deprecate all `executeGoal` methods and replace it by one `execute()` without arguments, in fact it is next command line argument ... but in next change if we wish

Reply:
> I saw it ... I thinking to deprecate all `executeGoal` methods and replace it by one `execute()` without arguments, in fact it is next command line argument ... but in next change if we wish

I absolutely support this position!

Reply:
Can you file an issue about that?

Reply:
> Can you file an issue about that?

Not yet

Reply:
[MSHARED-1128](https://issues.apache.org/jira/browse/MSHARED-1128)